metadata/IdCards__1__metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "changeLog": "Model: 21.5.5",
+  "cpu": 0,
+  "description": "Machine Learning model(available in Preview) for extracting commonly occurring data points from ID-Cards, including header fields and line items. Please see more details including supported languages and link to Activities guide in the About Licensing -> Document Understanding API Key section of the UiPath Automation Cloud Guide here: https://docs.uipath.com/automation-cloud/docs/about-licensing#section-document-understanding-api-key",
+  "displayName": "IDCards",
+  "gpu": 1,
+  "inputDescription": "ML Skills deployed using this ML Package can be integrated into RPA workflows using the Machine Learning Extractor activity in the Official feed or the MLModel activity in Connect feed. File formats accepted include pdf, tiff, jpg or png files. The activities also require an API Key input which you need to obtain from your UiPath Automation Cloud account, in the Licenses -> Other Services view.",
+  "inputType": "JSON",
+  "memory": 0,
+  "mlPackageLanguage": "PYTHON37_DU",
+  "name": "IDCards",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "GPU",
+  "projectId": "[project-id]",  
+  "retrainable": true,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v20.7.1",
```

Comment:
@irinel-birsan-uipath this should atleast be 20.10.1.2, and maybe 21.4. Language version wasn't supported till 20.10.1

Reply:
Yes, my bad, copy-paste from wrong place. Will do